src/aioswitcher/device/__init__.py

```diff
@@ -36,25 +36,31 @@ class DeviceCategory(Enum):
 class DeviceType(Enum):
     """Enum for relaying the type of the switcher devices."""
 
-    MINI = "Switcher Mini", "030f", 1, DeviceCategory.WATER_HEATER
-    POWER_PLUG = "Switcher Power Plug", "01a8", 1, DeviceCategory.POWER_PLUG
-    TOUCH = "Switcher Touch", "030b", 1, DeviceCategory.WATER_HEATER
-    V2_ESP = "Switcher V2 (esp)", "01a7", 1, DeviceCategory.WATER_HEATER
-    V2_QCA = "Switcher V2 (qualcomm)", "01a1", 1, DeviceCategory.WATER_HEATER
-    V4 = "Switcher V4", "0317", 1, DeviceCategory.WATER_HEATER
-    BREEZE = "Switcher Breeze", "0e01", 2, DeviceCategory.THERMOSTAT
-    RUNNER = "Switcher Runner", "0c01", 2, DeviceCategory.SHUTTER
-    RUNNER_MINI = "Switcher Runner Mini", "0c02", 2, DeviceCategory.SHUTTER
+    MINI = "Switcher Mini", "030f", 1, DeviceCategory.WATER_HEATER, False
+    POWER_PLUG = "Switcher Power Plug", "01a8", 1, DeviceCategory.POWER_PLUG, False
+    TOUCH = "Switcher Touch", "030b", 1, DeviceCategory.WATER_HEATER, False
+    V2_ESP = "Switcher V2 (esp)", "01a7", 1, DeviceCategory.WATER_HEATER, False
+    V2_QCA = "Switcher V2 (qualcomm)", "01a1", 1, DeviceCategory.WATER_HEATER, False
+    V4 = "Switcher V4", "0317", 1, DeviceCategory.WATER_HEATER, False
+    BREEZE = "Switcher Breeze", "0e01", 2, DeviceCategory.THERMOSTAT, False
+    RUNNER = "Switcher Runner", "0c01", 2, DeviceCategory.SHUTTER, False
+    RUNNER_MINI = "Switcher Runner Mini", "0c02", 2, DeviceCategory.SHUTTER, False
```

Comment:
I'm just making sure we aren't missing anything. All the current devices are set to `False.` I assume new devices contributed in the next steps of this work will be `True`, right?

Reply:
You are right. Switcher Runner S11 will be True.